query/sources/impl/src/main/java/org/codice/ddf/admin/sources/opensearch/discover/DiscoverOpenSearchSource.java

```diff
@@ -103,4 +105,12 @@ public OpenSearchSourceConfigurationField getReturnType() {
     private void setOpenSearchSourceUtils(OpenSearchSourceUtils openSearchSourceUtils) {
         this.openSearchSourceUtils = openSearchSourceUtils;
     }
+
+    @Override
+    public Set<String> getFunctionErrorCodes() {
```

Comment:
❓ Just thought of this as I hit this, but this will currently return the super errors (MISSING_REQUIRED_FIELD in this case) and the errors that happen during execution. It seems this should actually be union of the super errors, execution errors, and function argument errors?